nwb_conversion_tools/nwbconverter.py

```diff
@@ -56,7 +61,7 @@ def get_metadata_schema(self):
         """Compile metadata schemas from each of the data interface objects."""
         metadata_schema = get_root_schema()
         metadata_schema['properties'] = dict(
-            NWBFile=get_schema_from_hdmf_class(NWBFile),
+            NWBFile=get_schema_for_NWBFile(),
```

Comment:
why define this explicitly instead of generating it?

Reply:
The general function `get_schema_from_docval()` will include many fields for the NWBFile object schema that will not be used by the converter nor the GUI. It would end up cluttering the NWBFile form with many fields that will not be used.
I haven't found yet a programmatic way of skipping these, while preserving the possibility that they might be used as fields for other objects (such as TimeIntervals or DynamicTable). This is a dirty temporary solution. 

![image](https://user-images.githubusercontent.com/24541631/95835237-43b4fc00-0d3e-11eb-842d-9891cfab1d9b.png)

Reply:
What about adding a `skip` arg to `get_schema_from_hdmf_class` that's a list of names of the arguments to skip